packages/carbon-web-components/src/components/accordion/accordion-item-skeleton.ts

```diff
@@ -21,13 +21,14 @@ import { carbonElement as customElement } from '../../globals/decorators/carbon-
 class CDSAccordionItemSkeleton extends LitElement {
   render() {
     return html`
-      <span class="${prefix}--accordion__heading">
+      <span class="${prefix}--accordion__heading" part="expando">
```

Comment:
Need to document this and the other `part` attribute below with `@csspart` annottations on the class.